Comments/ClientApp/src/main.scss

```diff
@@ -24,7 +24,6 @@ $nice-font-base-path: "/consultations/fonts/";
 @import "./components/CommentBox/CommentBox";
 @import "./components/PhaseBanner/PhaseBanner";
 @import "./components/Selection/Selection";
-@import "./components/Footer/Footer";
```

Comment:
@suegarner we should also delete the actual file https://github.com/nhsevidence/consultations/blob/master/Comments/ClientApp/src/components/Footer/Footer.scss shouldn't we?

Reply:
Bah, had excluded it not deleted it. Done